lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -116,6 +116,99 @@ class CategoryRenderer {
   }
 
   /**
+   * @param {!ReportRenderer.AuditJSON} audit
+   * @param {number} scale
+   * @return {!Element}
+   */
+  _renderPerfHintAudit(audit, scale) {
+    const extendedInfo = /** @type {!CategoryRenderer.PerfHintExtendedInfo}
```

Comment:
ha, yeah +1 to keeping an object that is purely informational with no rendering awareness, which extendedInfo should be once we rip out formatters and whatnot. I see a difference between "renderer-aware" and "used by renderer" though. The former is bad and the latter seems fine in the short-term.